projects/packages/backup/src/js/hooks/useConnection.js

```diff
@@ -33,7 +33,15 @@ export default function useConnection() {
 		apiFetch( { path: '/jetpack/v4/backup-promoted-product-info' } ).then( res => {
 			setPrice( res.cost / 12 );
 			if ( res.introductory_offer ) {
-				setPriceAfter( res.introductory_offer.cost_per_interval / 12 );
+				const {
```

Comment:
Since this code is a replica of the other one checking for an intro discount, I wonder if we could make this an external function called "getDiscountedPrice" or something like that?